src/main/java/com/redhat/devtools/lsp4ij/operations/color/ColorInlayPresentation.java

```diff
@@ -62,7 +63,7 @@ public void paint(@NotNull Graphics2D g, @NotNull TextAttributes textAttributes)
         g.fillRect(x, y, getSquareDimension(), getSquareDimension());
 
         // Draw a border
-        g.setColor(isDarkTheme() ? Color.WHITE : Color.BLACK);
+        g.setColor(Gray.x00.withAlpha(40));//Same border color as com.intellij.util.ui.ColorIcon

```

Comment:
Makes it consistant with IJ proper and completion of color items:

<img width="210" alt="Screenshot 2024-01-09 at 17 08 17" src="https://github.com/redhat-developer/lsp4ij/assets/148698/6ec09c05-5b99-49fb-a4dc-a2e4ffac9b0d">